content/about.md

```diff
@@ -2,6 +2,38 @@
 title: About
 ---
 
-## About
+# About
 
-Help us to make _free communication_ possible and [contribute](https://invent.kde.org/melvo/xmpp-providers) to this project!
+XMPP (E**x**tensible **M**essaging and **P**resence **P**rotocol) is like a common language your and your chat partner's app speak to each other.
+Similar to email, there is a service provider managing your account and messages.
+That provider speaks XMPP too.
+
+But some XMPP providers support functionalities that other providers do not.
+Some support registrations via your app and some only via their websites.
+Some provide more space for sharing media than others or store them longer than others.
+The **XMPP Providers** makes it easy to find a suitable provider.
+
+All properties of the included providers have references in order to verify and maintain them.
+Therefore, it is important that providers add corresponding statements to their websites.
+
+This website uses XMPP Providers to generate its provider listing, show the detail pages for individual providers and explain how a provider could improve its category.
+Help us to make _free communication_ possible and [contribute](https://invent.kde.org/melvo/xmpp-providers/-/blob/master/CONTRIBUTING.md) to this project!
+
+## History
+
+XMPP Providers arose from [Kaidan](https://www.kaidan.im)'s onboarding introduced in [Kaidan 0.5](https://www.kaidan.im/2020/04/06/kaidan-0.5.0/).
+The goal was to speed up the onboarding process by making registrations as simple as possible.
```

Comment:
I think here its okay to say something alike "From here the authors, names.., developed and reviewed meadures to bring the project to the next level."

Reply:
I think we should also say that automated onboarding requires qualified server to ensure a good user experience. Furthermore, it provides more transparency and review of quality.